src/main/ngapp/src/app/app-routing.module.ts

```diff
@@ -11,15 +11,17 @@ const routes: Routes = [
   {
     path: 'login',
     component: LoginComponent,
-    canActivate: [AuthGuard],
+    canActivate: [AuthGuard]
   },
   {
     path: 'register',
     component: RegisterComponent,
+    pathMatch: 'full'
```

Comment:
why do we have ```pathmatch: full``` here? Should it not just be on the chat path since it is the default landing page if a user is logged in?

Reply:
It basically just ensures that we get that whole url. It doesn't actually affect much